components/Common/NodejsLogo/index.tsx

```diff
@@ -0,0 +1,30 @@
+import type { FC } from 'react';
+
+import NodejsDark from '@/components/Icons/Logos/NodejsDark';
+import NodejsDarkPride from '@/components/Icons/Logos/NodejsDarkPride';
+import NodejsLight from '@/components/Icons/Logos/NodejsLight';
+import NodejsLightPride from '@/components/Icons/Logos/NodejsLightPride';
+
+import style from './index.module.css';
+
+type NodejsLogoProps = {
+  isPrideEnabled?: boolean;
```

Comment:
Maybe something like
`variant: 'default' | 'pride'`
would fit better than booleans, considering that we may have more logos in the future.